### ISSUE TITLE
Feature request: overwriting spin-structure should not overwrite other parameters

### ISSUE THREAD

**Issue by SpiritSuperUser** [spirit-code/spirit#535]
While it is good to copy the whole system and all chosen parameters when one inserts a **new** image from a copied one, I think the more natural way in case one **replaces** a spin-structure in an **existing** image would be that only the spin-structure is inserted while all other parameters should remain as set in that image before. Having both optional, eg. by having another short-cut, might be an option in case not everyone agrees with my wanted default behaviour.

**Comment by GPMueller** [comment]
I agree this could be useful, but I believe it should not be the default. The question is what the shortcut should be. Maybe `shift+cmd+v`?

**Comment by SpiritSuperUser** [comment]
I would be fine with `shift+cmd+v`.
I, however, would change the default behavior and use the new shortcut for the old one.
